scripts/update-remote-wn-client

```diff
@@ -204,6 +189,16 @@ def rsync_upload(local_dir, remote_user, remote_host, remote_dir, ssh_key=None):
         raise Error("Error renaming remote directories: %s" % e)
 
 
+def setup_osg_ca_scripts_dir(osg_ca_scripts_dir):
+    subprocess.check_call([
+        "git", "clone", "-b", OSG_CA_SCRIPTS_BRANCH, OSG_CA_SCRIPTS_REPO, osg_ca_scripts_dir
+    ])
+    with chdir(osg_ca_scripts_dir):
+        os.symlink(".", "usr")
+        os.mkdir("etc/osg")
+        os.symlink("../osg-update-certs.conf", "etc/osg/osg-update-certs.conf")
```

Comment:
So just to confirm - `etc/osg-update-certs.conf` is the real file and `etc/osg/osg-update-certs.conf` is a symlink that points to it?

And the `usr` symlink is kinda there to make the `usr/` prefix optional?